marklogic-data-hub-central/ui/e2e/setup.sh

```diff
@@ -20,6 +22,7 @@ if $DHS
 then
         env=dhs
         perl -i -pe"s/mlHost=.*/mlHost=$mlHost/g" gradle-dhs.properties
+        ./gradlew -i mldeployusers -PenvironmentName=$env -PmlUsername=$mlAdminUsername -PmlPassword=$mlAdminPassword
         ./gradlew hubDeploy -PenvironmentName=$env --info --stacktrace
 else
         ./gradlew mlDeploy -PmlUsername=admin -PmlPassword=admin --info --stacktrace
```

Comment:
If we are requiring mlAdminUsername and mlAdminPassword for both dhs and local, should we be using those variables here as well?

Reply:
Right, thanks @ryanjdew will make that change